docs/src/get-started/introduction.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Introduction
+weight: -10
+description: |
+  Get started with Platform.sh. Start by learning why you should be interested.
+---
+
+Platform.sh is a cloud platform for responsibly building, running, and scaling fleets of websites and applications.
+It enables you to run your web apps in the cloud with productive and consistent development and testing workflows.
+
+You spend your time creating amazing experiences, not managing infrastructure.
+
+Get started with Platform.sh by following this guide.
+
+## Why
+
+The process of testing, deploying, and monitoring web applications (often grouped together under the term DevOps)
+can be complicated.
+You want to scale your offerings while maintaining consistency and reliability across your applications.
+
+Platform.sh simplifies your development workflows and helps you automate manual tasks across a fleet of websites.
+You can create build images you can test and then deploy with confidence,
+knowing the changes work the same in production as they have in your development process.
+
+See a more detailed overview of Platform.sh and how it can fit within your workflow in this video:
+
+{{< youtube ny2YeD6Qt3M >}}
+
+## What you need
+
+You need a few things before you can start creating projects.
+
+### A Platform.sh account
+
+To get started, you need a Platform.sh account.
+You can start with a free 30-day trial that should contain everything you need to complete this guide.
+
+If you don't have an account yet, [register for a account](https://auth.api.platform.sh/register).
+You can use an email address or a GitHub, Google, Bitbucket, or GitLab account.
+
+### Git
+
+GitOps is a term for workflows that use the Git version control software to manage infrastructure.
+It enables you to use familiar developer tools to manage Continuous Deployment in the cloud.
+
+Use the same version control system for your infrastructure as for your app development.
+You get a history of changes with an audit log.
+
+Git is at the center of work with Platform.sh.
+That's why each step in this guide is connected to a common Git command.
+
+Make sure your computer has [Git installed](https://git-scm.com/downloads).
+
+### CLI
+
+To facilitate working with Platform.sh, you can use the Platform.sh command-line interface (CLI).
+This lets you carry out various actions from a terminal.
+
+To install the CLI, you need:
+
+* [PHP 5.5.9+](https://www.php.net/manual/en/install.php) with the following extensions installed: `curl`, `json`, `mbstring`, `pcre`, and `phar`
+* A Bash-like shell
+
+  On Windows, the best way to get Bash is through [Windows Subsystem for Linux](https://msdn.microsoft.com/en-gb/commandline/wsl/about).
+
+{{% cli-installation %}}
+
+## Choose your stack
```

Comment:
In general, the tutorial should link to other documentation as little as possible. The idea is to try to focus on completing the flow rather than understanding the details.

Also, the idea is to replace the stacks documentation as a separate thing.